package.json

```diff
@@ -45,12 +45,13 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.5.0",
     "babelify": "^7.2.0",
-    "browserify": "^14.3.0",
+    "browserify": "^14.4.0",
     "budo": "^10.0.3",
-    "eslint": "^3.13.1",
-    "prettier": "^1.2.2",
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "eslint": "^4.1.1",
+    "prettier": "^1.5.2",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-dom-factories": "^1.0.0"
```

Comment:
But the react/react-dom are still okay right? This is about prettier and anything else I upgraded. I'll change those back.